Configuration/PageTs/defaultUploadFolders.ts

```diff
@@ -8,4 +8,7 @@ default_upload_folders {
     # You can set a folder for the whole table of for a specific field of that table
     tx_news_domain_model_news.fal_related_files = news_downloads
     tx_news_domain_model_news.fal_media = news_media
+
+    # You can set a fallback for all tables
+    DefaultForAllTables = 1:myDefaultUploadFolderForThisPartOfTheTree
```

Comment:
Can you change this so it starts lower case like all other settings: `defaultForAllTables`.